### DISCUSSION TITLE
[Bug]

### DISCUSSION THREAD

**Discussion by lemon-Star** [Chia-Network/chia-blockchain#9722] [Support]
### What happened?

您好：
   我的钱包地址莫名多了一笔转入交易，但是我的钱包中原有的2.27个XCH币不见了，我通过区块查询，有一个可疑 的问题，同一个区块下有两笔转账，一个是给我的账户转入0.00005个XCH，另一个是给其他账户转入了我的余额，这是不是一个BUG呢
![XCH](https://user-images.githubusercontent.com/58968187/146925074-511f967f-a379-4efe-8fea-e861795842dd.jpg)

这是我的钱包地址：xch1vvd4zpdxh7vs7ccua7g2gmlvvsz5wmdqt0r6jz6kty3ysxfywfzs4leklh
这是可疑的hash：0x535c3d1f042f858e7c734a17a864c22980c01623be3daf9b4510b2d614cd041a

### Version

1.2.11

### What platform are you using?

Windows

### What ui mode are you using?

GUI

**Comment by ruixingchen** [comment]
兄弟，你这是被盗号了，换个助记词和安全的环境吧带兄弟
